### ISSUE TITLE
[Feature Request] Streamlabs Currency Support

### ISSUE THREAD

**Issue by Raxxion** [iChun/ContentCreatorIntegration-IssuesAndDocumentation#16]
Adding support for Streamlabs Currency system to trigger CCI Events

**Comment by gaz8203** [comment]
This would be a great addition!

**Comment by iChun** [comment]
Can someone please explain to me what the streamlabs currency system is. Is this the one on on the chatbot?

**Comment by Malacara** [comment]
> Can someone please explain to me what the streamlabs currency system is. Is this the one on on the chatbot?

Yes and currently it works, I have spent months configuring this mod for many streamers and I have made it work even with the economy of twitch, streamlabs and more. Maybe I can prepare a guide or help you with the wiki iChun.

**Comment by iChun** [comment]
@Malacara feel free to contribute to the documentation however you can, you're welcome to even make a separate area as a guide or something like that. The docs are very very incomplete right now and I'm just frankly too busy IRL to do anything about it
